SPECS-EXTENDED/stunnel/stunnel.spec

```diff
@@ -1,47 +1,60 @@
-Vendor:         Microsoft Corporation
-Distribution:   Mariner
+%bcond_with libwrap
 # Do not generate provides for private libraries
 %global __provides_exclude_from ^%{_libdir}/stunnel/.*$
 
-
-%bcond_with libwrap
-
-
-
-
-Summary: A TLS-encrypting socket wrapper
-Name: stunnel
-Version: 5.56
-Release: 8%{?dist}
-License: GPLv2
-URL: http://www.stunnel.org/
-Source0: https://www.stunnel.org/downloads/stunnel-%{version}.tar.gz
-Source1: https://www.stunnel.org/downloads/stunnel-%{version}.tar.gz.asc
-Source2: Certificate-Creation
-Source3: sfinger.xinetd
-Source4: stunnel-sfinger.conf
-Source5: pop3-redirect.xinetd
-Source6: stunnel-pop3s-client.conf
-Source7: stunnel@.service
-Patch0: stunnel-5.50-authpriv.patch
-Patch1: stunnel-5.50-systemd-service.patch
-Patch3: stunnel-5.56-system-ciphers.patch
-Patch4: stunnel-5.56-coverity.patch
-Patch5: stunnel-5.56-default-tls-version.patch
-Patch6: stunnel-5.56-curves-doc-update.patch
+Summary:        A TLS-encrypting socket wrapper
+Name:           stunnel
+Version:        5.70
+Release:        1%{?dist}
+License:        GPLv2
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            http://www.stunnel.org/
+Source0:        https://www.stunnel.org/downloads/stunnel-%{version}.tar.gz
+Source2:        Certificate-Creation
+Source3:        sfinger.xinetd
+Source4:        stunnel-sfinger.conf
+Source5:        pop3-redirect.xinetd
+Source6:        stunnel-pop3s-client.conf
+Source7:        stunnel@.service
+# Apply patch stunnel-5.50-authpriv.patch
```

Comment:
Nit, these "apply patch xyz" comments are not particularly useful. If the purpose of these patches can be easily discovered, it would be great to add those details.

Reply:
Sure, will do.